# RET: 1
# ERR: {self}:9:12:    stuff(4, z)
# ERR: {ssss}                 ^ error: Passing >= 2 arguments to function, but it expects 1.
def int stuff(int x):
    return 0

def int main():
    z = "hi"
    stuff(4, z)
